out/addresses.json

```diff
@@ -1,22 +1,24 @@
 {
-  "GEM": "0x7ba25f791fa76c3ef40ac98ed42634a8bc24c238",
-  "GOV": "0x1c3ac7216250edc5b9daa5598da0579688b9dbd5",
-  "PIP": "0xb7092ee7a8c4c85431962662310bbdcd4fd519e9",
-  "PEP": "0xc0ee05307ae4a5316f34874a3525d10c94b3c217",
+  "ETH_RPC_URL": "127.0.01:2000",
+  "ETH_FROM": "0x16fb96a5fa0427af0c8f7cf1eb4870231c8154b6",
+  "GEM": "0x51a254aedc853d48616cefcdd48e57165de66abb",
```

Comment:
when these addresses change, they should be accompanied by a corresponding change to the default snapshot so that anyone pulling down the branch can use the snapshot and the outputs without having to `dapp create` anything

Reply:
done for both PR's